angular/projects/spark-angular/src/lib/components/sprk-tabbed-navigation/sprk-tabbed-navigation.stories.ts

```diff
@@ -30,7 +30,16 @@ export default {
       SprkTabbedNavigationPanelDirective,
       SprkTabbedNavigationTabDirective,
     },
-    info: markdownDocumentationLinkBuilder('tabs'),
+    info: `
+${markdownDocumentationLinkBuilder('tabs')}
+- The Tabbed Navigation component makes use of the
```

Comment:
I think we decided on calling it "Tabs" instead of Tabbed Navigation.

Reply:
We did, but it is technically <sprk-tabbed-navigation> is angular. So I didn't change it when referring to the component directly.